test/e2e/specs/align-hook.test.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Internal dependencies
+ */
+import {
+	newPost,
+	insertBlock,
+	getEditedPostContent,
+	setCodeEditorContent,
+	switchToEditor,
+	getAllBlocks,
+	selectBlockByClientId,
+} from '../support/utils';
+import { activatePlugin, deactivatePlugin } from '../support/plugins';
+
+describe( 'Align Hook Works As Expected', () => {
+	beforeAll( async () => {
+		await activatePlugin( 'gutenberg-test-align-hook' );
+	} );
+
+	beforeEach( async () => {
+		await newPost();
+	} );
+
+	afterAll( async () => {
+		await deactivatePlugin( 'gutenberg-test-align-hook' );
+	} );
+
+	const getAlignmentToolbarLabels = async () => {
+		const buttonLabels = await page.evaluate( () => {
+			const buttons = [];
+			document.querySelectorAll(
+				'.editor-block-toolbar button[aria-label^="Align"]'
+			).forEach(
+				( button ) => {
+					buttons.push( button.getAttribute( 'aria-label' ) );
+				}
+			);
+			return buttons;
+		} );
+		return buttonLabels;
+	};
+
+	const showsTheExpectedButtons = ( blockName, buttonLabels ) => {
```

Comment:
How do you feel about renaming those methods that create a test from:
`showsTheExpectedButtons` | `doesNotApplyAlignmentByDefault` | `correctlyAppliesAndRemovesAlignment`
to
`createShowsTheExpectedButtonsTest` | `createDoesNotApplyAlignmentByDefaultTest` | `createCorrectlyAppliesAndRemovesAlignmentTest`.